File: FCUBEAPP/src/app/models/dieselstatementmodel.ts

import { Dieselstmtdtlsmodel } from "./dieselstmtdtlsmodel";

export class Dieselstatementmodel {
    masterID: string = "";
    dfVendor: string = "";
    vendor: string = "";
    billStmtNo: string = "";
    billStmtDate: string = "";
    fromDate: string = "";
    toDate: string = "";
    findocid: string = "";
    location: string = "";
    transType:string = "";
    rate: string = "";
    statementFlag: string = "";
    remarks: string = "";
    totalDslLtrs: string = "";
    grossDslAmt: string = "";
    discRateLtr: string = "";
    discAmt: string = "";
    totalDslAmt: string = "";
    tdsRate: string = "";
    tdsAmt: string = "";
    totalCashAdv: string = "";
    totalNetAmount: string = ""; 
    branchCode: string = "";
    yearId: string = "";
    driverId: string = "";
    createdBy: string = "";
    createdDate: string = "";
    modifiedBy: string = "";
    modifiedDate: string = "";
    loggedInUser: string = "";
    dieselStatementListData: Dieselstatementsearchmodel[]=[];
    dieselStmtDtlsList: Dieselstmtdtlsmodel[]=[];
}

export class Dieselstatementsearchmodel {
    pmtId: string = "";
    branch: string = "";
    pmtDate: string = "";
    vehicleNo: string = "";
    hsdAdvType: string = "";
    transDesc: string = "";
    qtyLtrs: string = "";
    ratePerLtr: string = "";
    amountPaid: string = "";
    remarks: string = "";
    selected: boolean = false;
}

File: OMKARAUTOAPI/FleetTrans/Models/DieselStatementModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FleetTrans.Models
{
    public class DieselStatementModel
    {
        public string? MasterID { get; set; }
        public string? DfVendor { get; set; }
        public string? Vendor { get; set; }
        public string? BillStmtNo { get; set; }
        public string? BillStmtDate { get; set; }
        public string? FromDate { get; set; }
        public string? ToDate { get; set; }
        public string? Findocid { get; set; }
        public string? Location { get; set; }
        public string? Rate { get; set; }
        public string? StatementFlag { get; set; }
        public string? Remarks { get; set; }
        public string? TotalDslLtrs { get; set; }
        public string? GrossDslAmt { get; set; }
        public string? DiscRateLtr { get; set; }
        public string? DiscAmt { get; set; }
        public string? TotalDslAmt { get; set; }
        public string? TdsRate { get; set; }
        public string? TdsAmt { get; set; }
        public string? TotalCashAdv { get; set; }
        public string? TotalNetAmount { get; set; }      
        public string? BranchCode { get; set; }
        public string? YearId { get; set; }
        public string? DriverId{ get; set; }
        public string? FromLoc { get; set; }
        public string? ToLoc { get; set; }
        public string? CreatedBy { get; set; }
        public string? CreatedDate { get; set; }
        public string? ModifiedBy { get; set; }
        public string? ModifiedDate { get; set; }
        public string? LoggedInUser { get; set; }
        public List<DieselStatementSearchModel> DieselStatementListData { get; set; }
        public List<DieselStmtDtlsModel> DieselStmtDtlsList { get; set; }
    }
}
